L/libevdev/build_tarballs.jl

```diff
@@ -0,0 +1,40 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libevdev"
+version = v"1.11.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://gitlab.freedesktop.org/libevdev/libevdev", "8855f1ac59a69d5bdff51e3f8980697f0127c270"),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/libevdev/
+atomic_patch ../patches/link_rt.patch
+mkdir build
+cd build
+
+PKG_CONFIG_SYSROOT_DIR="" meson -D tests=disabled -D documentation=disabled ../ --cross-file="${MESON_TARGET_TOOLCHAIN}"
```

Comment:
Is `PKG_CONFIG_SYSROOT_DIR=""` actually needed here?

Reply:
Likely not, I think I did have a host dependency before, but that wasn't needed anymore after disabling tests and documentation.

```suggestion
meson -D tests=disabled -D documentation=disabled ../ --cross-file="${MESON_TARGET_TOOLCHAIN}"
```